abbreviato.gemspec

```diff
@@ -1,4 +1,4 @@
-$:.push File.expand_path("../lib", __FILE__)
+$:.push File.expand_path("lib", __dir__)
```

Comment:
Changed because of Rubocop complaint.

```
Inspecting 10 files
..C.......

Offenses:

abbreviato.gemspec:1:14: C: Style/ExpandPathArguments: Use expand_path('lib', __dir__) instead of expand_path('lib', __FILE__).
$:.push File.expand_path("lib", __FILE__)
             ^^^^^^^^^^^
```